content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -255,3 +255,61 @@ spec:
     tls:
       mode: DISABLE
 {{< /text >}}
+
+## Migrating the `RbacConfig` to `ClusterRbacConfig`
+
+The `RbacConfig` has a [bug](https://github.com/istio/istio/issues/8825). The bug reduces the scope of
+the object to be namespace-scoped in some cases. The `ClusterRbacConfig` follows the exact same
+specification as the `RbacConfig` but with correct cluster scope implementation.
+
+1. Replace the custom resource of kind `RbacConfig` with kind `ClusterRbacConfig` if you use
+   [Istio authorization](/docs/concepts/security/#authorization) with custom resource of kind `RbacConfig`:
+
+    A script is provided to automate the migration for you. Depending on if you have the [Istio installation package](/docs/setup/kubernetes/download-release):
+
+    - If you have the Istio installation package
+
+        Move to the Istio package directory and run the script with the following command:
+
+        {{< text bash >}}
+        $ ./tools/convert_RbacConfig_to_ClusterRbacConfig.sh
+        {{< /text >}}
+
+    - If you don't have the Istio installation package
+
+        Download and run the script directly with the following command:
+
+        {{< text bash >}}
+        $ curl -L https://raw.githubusercontent.com/istio/istio/master/tools/convert_RbacConfig_to_ClusterRbacConfig.sh | sh -
+        {{< /text >}}
+
+The script will get existing `RbacConfig` and apply a `ClusterRbacConfig` with same specification as
+Kubernetes doesn't allow to change the kind of a custom resource once it's created.
+
+For example, if you have a `RbacConfig` like the following one:
+
+{{< text yaml >}}
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: RbacConfig
+metadata:
+  name: default
+spec:
+  mode: 'ON_WITH_INCLUSION'
+  inclusion:
+    namespaces: ["default"]
+{{< /text >}}
+
+The script will create the following `ClusterRbacConfig`:
+
+{{< text yaml >}}
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ClusterRbacConfig
+metadata:
+  name: default
+spec:
+  mode: 'ON_WITH_INCLUSION'
+  inclusion:
+    namespaces: ["default"]
+{{< /text >}}
+
+The script will wait for a few seconds to let the `ClusterRbacConfig` to take effect and then delete the `RbacConfig`.
```

Comment:
This line contains two steps:
1. Wait for a few seconds to let the configuration take effect.
1. Delete the previous configuration file:
   <The command to remove the file with the exact path and filename of the file that needs to be removed.

Reply:
The script will do all these steps (migrate, wait and delete), for user there is only one step that is to run the script.

Reply:
I applaud the fact the script does all those things! A slight reword might be all you need @yangminzhu:
The script automates the following operations: 
1. Create the cluster RBAC configuration file.
2. Apply the configuration and wait for a few seconds to let the configuration take effect.
3. Delete the previous RBAC configuration file.